Dockerfile

```diff
@@ -12,10 +14,11 @@ RUN INSTALL_PKGS=" \
     yum clean all && \
     mkdir /tmp/_working_dir && \
     chmod og+w /tmp/_working_dir
-COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
-COPY scripts/* /usr/bin/scripts/
-COPY files/ /usr/bin/files/
-ADD controller-manifests /manifests
+COPY --from=builder _output/bin/cluster-logging-operator /usr/bin/
+COPY --from=builder scripts/* /usr/bin/scripts/
+COPY --from=builder files/ /usr/bin/files/
```

Comment:
this is going to break some of the components like rsyslog...
https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/rsyslog.go#L54

Reply:
Also, the COPY target has been moved to /usr/share/logging/ in master-post-release...
https://github.com/openshift/cluster-logging-operator/blob/master-post-release/Dockerfile#L18

Reply:
why would it break them?  the builder image has all the source already.  This just copies it from their instead